modules/launchd/default.nix

```diff
@@ -23,7 +23,10 @@ let
       cmd = config.command;
       env = config.environment // optionalAttrs (config.path != "") { PATH = config.path; };
 
-    in
+      wait4Command = pkgs.writeShellScript "wait4command"
+        "/bin/wait4path $(realpath $(type -P \"$1\")) && exec \"$@\"";
```

Comment:
```suggestion
```

This won't work as `pkgs.writeShellScript` will give you a shell script inside `/nix/store` which might not be available